src/views/details/details.js

```diff
@@ -0,0 +1,97 @@
+myApp.DetailsPage = function(i) {
+    const evData = localStorage.getItem("evaluationsData");
+    const tableData = JSON.parse(evData);
+    this.markup = `
+    ${myApp.DetailsTable(tableData)}
+    `
+}
+
+myApp.DetailsTable = function(tableData) {
+    return `
+    
+             <p>Candidate Info:</p>
+             <table class="modal-table">
+                 <tr>
+                     <td class="modal-table-data">Candidate Name</td>
+                     <td class="modal-table-data">${tableData[0].candidateInfo[0].value}</td>
+                 </tr>
+                 <tr>
+                     <td class="modal-table-data">Interviewer</td>
+                     <td class="modal-table-data">${tableData[0].candidateInfo[1].value}</td>
+                 </tr>
+                 <tr>
+                     <td class="modal-table-data">Interview Date</td>
+                     <td class="modal-table-data">${tableData[0].candidateInfo[2].value}</td>
+                 </tr>
+                 <tr>
+                     <td class="modal-table-data">Should the candidate be hired?</td>
+                     <td class="modal-table-data">${tableData[0].comments[0].value}</td>
+                 </tr>
+                 <tr>
+                     <td class="modal-table-data">General Impression</td>
+                     <td class="modal-table-data">${tableData[0].comments[1].value}</td>
+                 </tr>
+                 <tr>
+                     <td class="modal-table-data">Workflow, Leadership & Softskills</td>
+                     <td class="modal-table-data">${tableData[0].comments[2].value}</td>
+                 </tr>                
+             </table>
+             <p>Candidate Evaluation:</p>
+             <table class="modal-table">
+             <tr>
+                 <td class="modal-table-data">OOP, Design Patterns</td>
+                 <td class="modal-table-data">${tableData[0].evaluation[0].value}</td>
```

Comment:
I think I've already previously commented on this, but having everything rely on its position inside a single array can quickly come to bite you, should you ever need to add or remove a category.

Also, I firmly hold copying and pasting code as one of the capital sins of software development, because it makes it remarkably easy for issues to slip by unnoticed, or circumvent opportunities for code cleanup, abstraction, refactoring and generalization. Whenever you find yourself doing `Ctrl + C`, followed by `Ctrl + V`, know that I am watching and quietly sobbing...